client/index.html

```diff
@@ -356,25 +357,352 @@ <h1 class="title">Settings</h1>
+								<code>/query nick</code>
+							</div>
+							<div class="description">
+								<p>Send a private message to the specified user.</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/quit [message]</code>
+							</div>
+							<div class="description">
+								<p>
+									Disconnect from the current network with an optional message.
+								</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/raw message</code>
+							</div>
+							<div class="description">
+								<p>Send a raw message to the current IRC network.</p>
+								<p>Aliases: <code>/quote</code>, <code>/send</code></p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/slap nick</code>
+							</div>
+							<div class="description">
+								<p>Slap someone in the current channel with a trout!</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/topic newtopic</code>
+							</div>
+							<div class="description">
+								<p>Set the topic in the current channel.</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/voice nick [...nick]</code>
+							</div>
+							<div class="description">
+								<p>
+									Give voice (<code>+v</code>) to one or several users in the
+									current channel.
 								</p>
 							</div>
 						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/whois nick</code>
+							</div>
+							<div class="description">
+								<p>
+									Retrieve information about the given user on the current
+									network.
+								</p>
+							</div>
+						</div>
+
+						<h2>Keyboard Shortcuts</h2>
+
+						<h3>On Windows / Linux</h3>
+
+						<div class="help-item">
+							<div class="subject">
+								<kbd>Ctrl</kbd> + <kbd>↑</kbd> / <kbd>↓</kbd>
+							</div>
+							<div class="description">
+								<p>Switch to the previous/next window in the channel list</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>L</kbd>
+							</div>
+							<div class="description">
+								<p>Clear the current screen</p>
+							</div>
+						</div>
+
+						<h3>On macOS</h3>
+
+						<div class="help-item">
+							<div class="subject">
+								<kbd>⌘</kbd> + <kbd>↑</kbd> / <kbd>↓</kbd>
+							</div>
+							<div class="description">
+								<p>Switch to the previous/next window in the channel list</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<kbd>⌘</kbd> + <kbd>K</kbd>
+							</div>
+							<div class="description">
+								<p>Clear the current screen</p>
+							</div>
+						</div>
+
+						<h2>About The Lounge</h2>
+
+						<p class="about">
```

Comment:
I understand your concerns but here is my take on this:

- As this links to the release notes, the documentation and reporting a new issue, this seems more valuable to have it in the `Help` page than the `Settings` page. Even without those links, this is more informative than configurable, so still a bit more appropriate.
- I am going to add a bit of "help" value to it very soon as I'll be adding a couple links to new resources of the website.

Having it separate is reasonable but I think outside of the scope of this PR (and of the time I can reasonably spend on this project :D). One idea would be to have tabs in this page after we add similar tabs to the Settings page and `About` would be one tab.
But again, timeframe-wise, we're not there yet, and I'd argue there are much more pressing things I'd like to spend my time on instead 😅